STYLE.md

```diff
@@ -30,7 +30,8 @@
   fiasco](https://isocpp.org/wiki/faq/ctors#static-init-order-on-first-use) for
   how to best handle this.
 * API-level comments should follow normal Doxygen conventions. Use `@param` to describe
-  parameters, `@return <return-type>` for return values.
+  parameters, `@return <return-type>` for return values. Internal comments for
```

Comment:
It does increase the activation energy to comment by forcing people to document every arg/return. I might want to add a quick:

```
// foo does bar
int foo(char arg);
```

comment, I don't necessarily want to add full docs for return/args, but the documentation is better than nothing.